qiskit/algorithms/time_evolvers/variational/var_qte_result.py

```diff
@@ -0,0 +1,65 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Result object for varQTE."""
+from __future__ import annotations
+
+import numpy as np
+
+from qiskit.circuit import QuantumCircuit
+
+from ..time_evolution_result import TimeEvolutionResult
+
+from ...list_or_dict import ListOrDict
+
+
+class VarQTEResult(TimeEvolutionResult):
+    """The result object for the variational quantum time evolution algorithms.
+
+    Attributes:
+        evolved_state (QuantumCircuit): An evolved quantum state.
+        aux_ops_evaluated (ListOrDict[tuple[complex, complex]] | None): Optional list of
+            observables for which expected values on an evolved state are calculated at the
+            last timestep. These values are in fact tuples formatted as (mean, standard deviation).
+        observables (ListOrDict[tuple[np.ndarray, np.ndarray]] | None): Optional list of
+            observables for which expected on an evolved state are calculated at each timestep.
+            These values are in fact lists of tuples formatted as (mean, standard deviation).
+        times (np.array | None): Optional list of times at which each observable has been evaluated.
+        parameter_values (np.array | None): Optional list of parameter values obtained after
```

Comment:
The first four attributes above are from the parent class and documented there. Duplicating these here does seem like its awkward for maintenance etc so ideally it should just be the additional value supplied here I would have thought. I cannot see how this looks as it does not appear to be in the set of classes that are listed for the documentation and I guess it should be.